offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/DependencyManager.groovy

```diff
@@ -601,17 +613,17 @@ class DependencyManager {
             throw e
         }
 
-        /**CreateProductView copyProductView
+        CopyProductView copyProductView
         try{
-            copyProductView = new CreateProductView(copyProductViewModel,maintainProductController)
+            copyProductView = new CopyProductView(copyProductViewModel, maintainProductController)
         }catch(Exception e){
             log.error("Could not create ${CreateProductView.getSimpleName()} view.", e)
```

Comment:
```suggestion
            log.error("Could not create ${CopyProductView.getSimpleName()} view.", e)
```

Reply:
They call me the fastest copy paster in the west 🔫